policyengine_us/parameters/gov/states/sc/tax/income/subtractions/retirement_deduction/subtract_military.yaml

```diff
@@ -0,0 +1,16 @@
+description: South Carolina subtracts the survivors retirement deduction from the military retirement deduction if this is true.
+metadata:
+  unit: bool
+  label: South Carolina subtracts the survivors retirement deduction
+  reference: 
+  - title: Instructions for Form SC-1040 (line 3p)
+    href: https://dor.sc.gov/forms-site/Forms/IITPacket_2021.pdf#page=17
+  - title: Instructions for Form SC-1040 (line 3p)
+    href: https://dor.sc.gov/forms-site/Forms/IITPacket_2022.pdf#page=19
+  - title: SC-1040 2021
+    href: https://dor.sc.gov/forms-site/Forms/SC1040_2021.pdf#page=2
+  - title: SC-1040 2022
```

Comment:
does the law represent this in any way? If not please add a comment that this is purely based on the tax forms

Reply:
The law mentions this:
![image](https://github.com/PolicyEngine/policyengine-us/assets/113824411/074cd061-40f0-411a-aaa1-7e2374cc90c8)
this is the way we calculated for 2022. But it did not mention 2021. [Section 12-6-1171](https://www.scstatehouse.gov/code/t12c006.php) is about military retirement deductions. I will add the law reference and a comment.